src/Controller/Simulation.java

```diff
@@ -467,6 +467,23 @@ public String toggleRunning() {
         this.running = true;
         return "Simulation ON";
     }
+    
+    // Probably illegal to put that like this... [JM]
+    boolean LightAuto = true;
```

Comment:
Just move this to the top of the file with the other attributes.  And maybe initialise it to true inside the Simulation constructor.

Reply:
Added a commit that resolved this!